charts/nri-kube-events/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Kube Events
 name: nri-kube-events
-version: 1.3.2
+version: 1.3.3
```

Comment:
In the meanwhile a chart with 1.3.3 was released, can you bump it to 1.3.4?

Reply:
sure! let me fix this now